src/EditorFeatures/Test2/ReferenceHighlighting/VisualBasicReferenceHighlightingTests.vb

```diff
@@ -272,5 +272,27 @@ End Class
 
             VerifyHighlights(input)
         End Sub
+
+        <WorkItem(1904, "https://github.com/dotnet/roslyn/issues/1904")>
+        <WorkItem(2079, "https://github.com/dotnet/roslyn/issues/2079")>
```

Comment:
This test currently addresses the issue in 1904. Once 2079 is fixed, this will start failing. It should be fixed up by adding the {|Reference:symbol|} tags, where symbol denotes the alias symbol or its target. Then it will cover both 1904 and 2079. I cannot possibly make a test that touches 1904 but wont fail after 2079 is addressed. The only other way would be to add the workitem tags and keep this test disabled until 2079 is fixed. Which then opens up the possibility that this will remain disabled.